src/main/java/duke/UI.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+/**
+ * Encapsulates a class that contains methods for interacting with the user
+ */
+public class UI {
```

Comment:
Instead of UI (full caps), it'll match coding standards more if the class is refactored to 'Ui' (pascal case)!

Reply:
Fixed